init-tools.sh

```diff
@@ -186,6 +186,9 @@ ls "$__scriptpath/Tools/scripts/docker/"*.sh | xargs chmod +x
 
 "$__scriptpath/Tools/crossgen.sh" "$__scriptpath/Tools"
 
+echo "Patching Sdk to workaround issue https://github.com/dotnet/sdk/pull/1997"
+git apply --ignore-whitespace --whitespace=nowarn --directory="$__scriptpath" "$__scriptpath/patches/0001-Patch-Tools-for-sdk-pr-1997.patch"
```

Comment:
Is it reasonable to do this during `tools-local/init-build.proj` to avoid customizing this script? (With some semaphore in the Tools dir to avoid attempting it more than once?)

Reply:
@dagood and I discussed offline.  I will leave it in the init-tools.sh script since 1) it is very temporary and will go away on the next update of CLI version and 2) it can more easily participate in the Tools setup semaphore.